packages/dropdowns/src/elements/Menu/Menu.tsx

```diff
@@ -22,18 +22,23 @@ import {
 import { MenuContext } from '../../utils/useMenuContext';
 
 interface IMenuProps extends HTMLAttributes<HTMLUListElement> {
-  /** See Popper [documentation](https://popper.js.org/docs/v2/modifiers/) for details */
+  /** Sets the [Popper](https://popper.js.org/docs/v2/modifiers/) modifiers */
   popperModifiers?: Modifiers;
+  /** Determines if events are enabled */
```

Comment:
@austingreendev please provide a bit more context about what is going on here. This could use a better description.

Reply:
[Here is the full description provided by Popper.](https://popper.js.org/docs/v1/#Popper.Defaults.eventsEnabled) We might just want to use some version of this:

> Whether events (resize, scroll) are initially enabled

Reply:
Fixed in latest push.